modules/openapi-generator/src/main/resources/Java/libraries/vertx/ApiClient.mustache

```diff
@@ -680,54 +680,54 @@ public class ApiClient{{#jsr310}} extends JavaTimeFormatter{{/jsr310}} {
 
         private final Map<String, Authentication> authentications = new LinkedHashMap<>();{{#authMethods}}{{#isBasic}}{{#isBasicBasic}}
 
-        public void add{{#lambda.titlecase}}{{name}}{{/lambda.titlecase}}Authentication(String username, String password) {
+        public void add{{#lambda.titlecase}}{{#lambda.camelcase}}{{name}}{{/lambda.camelcase}}{{/lambda.titlecase}}Authentication(String username, String password) {
```

Comment:
thanks for the PR. we can go with this fix to start with

ideally, we should use a pascalcase lambda to avoid using double lambda.

Reply:
Ah, I figured there was a single lambda I could use but I didn't look hard enough. Thank you for reviewing!

Reply:
do you mind filing a follow-up PR to update it with pascalcase lambda?

Reply:
Sure, coming right up.

Reply:
thanks, you can add the pascalcase lambda based onhttps://github.com/OpenAPITools/openapi-generator/blob/edbb021aadae47dcfe690313ce5119faf77f800d/modules/openapi-generator/src/main/java/org/openapitools/codegen/templating/mustache/CamelCaseLambda.java

Reply:
Perhaps you were not asking me to add a new class, and I misunderstood. Never mind!

Reply:
ah right. I forgot about that one.

no need to add a new class then. sorry for the confusion.

Reply:
Here it is - https://github.com/OpenAPITools/openapi-generator/pull/18667/files